### PR TITLE
Darwin staging branch

### PR DESCRIPTION
**Pull request by matthewbauer**
Some commits on top of staging-next branch to fix issues with my #47678 pr.

### CODE REVIEW

File: pkgs/os-specific/darwin/apple-sdk/frameworks.nix [hunk 1]
```diff
@@ -82,9 +82,7 @@ with frameworks; with libs; {
   PCSC                    = [ CoreData ];
   PreferencePanes         = [];
   PubSub                  = [];
-  QTKit                   = [ CoreMediaIO CoreMedia MediaToolbox QuickTime VideoToolbox ];
```
Comment (LnL7) [inline pkgs/os-specific/darwin/apple-sdk/frameworks.nix]: Why did you remove a bunch of these? They are still in the SDK.

Comment (matthewbauer) [reply pkgs/os-specific/darwin/apple-sdk/frameworks.nix]: Are you sure? Maybe they’re in another download. I’ll find the the error i was getting...

Comment (LnL7) [reply pkgs/os-specific/darwin/apple-sdk/frameworks.nix]: I looked at `darwin.apple_sdk.sdk` which is the source IIRC.

Comment (matthewbauer) [reply pkgs/os-specific/darwin/apple-sdk/frameworks.nix]: Maybe just quicktime was removed?

Comment (LnL7) [reply pkgs/os-specific/darwin/apple-sdk/frameworks.nix]: Ah indeed, the QuickTime headers are no longer there.
